internal/tools/kops/cluster.go

```diff
@@ -79,11 +79,11 @@ func (c *Cmd) CreateCluster(name, cloud string, kopsRequest *model.KopsMetadataR
 // Example setValue: spec.kubernetesVersion=1.10.0
 func (c *Cmd) SetCluster(name, setValue string) error {
```

Comment:
There is another `SetInstanceGroup` method that should need this change: https://github.com/mattermost/mattermost-cloud/blob/master/internal/tools/kops/instance_groups.go#L221